### PR TITLE
When using seats, other guests (name, email) are no longer always listed on the success page

### LINKED ISSUE #5341: [CAL-311] When using seats, other guests (name, email) are always listed on the success page
Other guests should only be listed when "Share attendee information between guests" has been selected.

image

> From [SyncLinear.com](https://synclinear.com)

### PR DESCRIPTION
**Pull request by om-ray**
## What does this PR do?

This PR fixes the issue with other guests' information being shared even when the "Share attendee information between guests" checkbox hasn't been checked

Fixes #5341 

**Environment**: Staging(main branch) / Production

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Comment by om-ray** [comment]
@alannnc @PeerRich @emrysal Please review the PR!

File: apps/web/pages/success.tsx [hunk 1]
```diff
@@ -359,29 +359,32 @@ export default function Success(props: SuccessProps) {
...
-                              <p className="text-bookinglight">{bookingInfo.user.email}</p>
-                            </div>
-                          )}
-                          {!!eventType.seatsShowAttendees
-                            ? bookingInfo?.attendees
-                                .filter((attendee) => attendee.email === email)
-                                .map((attendee) => (
-                                  <div key={attendee.name} className="mb-3">
+                          <>
+                            {console.log(eventType.seatsShowAttendees)}
```
Comment (alannnc) [inline apps/web/pages/success.tsx:363]:
```suggestion
<>
```

Comment (alannnc) [inline apps/web/pages/success.tsx:363]:
```suggestion

```

**Comment by om-ray** [comment]
@alannnc Thank you so much for removing that log, I need to remember to remove those 😅

**Comment by alannnc** [comment]
Looks good! Let me just test it on the preview deploy and it should be merged soon. Thanks for your help.

**Comment by om-ray** [comment]
Of course! I'm happy to help!

**Review by alannnc — APPROVED** [review]

**Pull request merged**